include/sys/spa_impl.h

```diff
@@ -66,6 +66,7 @@ typedef struct spa_error_entry {
 	zbookmark_phys_t	se_bookmark;
 	char			*se_name;
 	avl_node_t		se_avl;
+	zbookmark_err_phys_t	se_zep;		/* not accounted in avl_find */
```

Comment:
By not accounted for you simply mean it's not used by the `spa_error_entry_compare` function during `avl_find()`.  Right?

Reply:
Right. Thank you for taking a look!

Reply:
I did it that way due to compatibility reasons.